maintenance_plan_meter_mode/models/maintenance.py

```diff
@@ -0,0 +1,129 @@
+from odoo import api, models, fields, _
+from datetime import timedelta
+from odoo.exceptions import ValidationError
+
+
+class MaintenancePlan(models.Model):
+    _inherit = 'maintenance.plan'
+
+    meter_mode = fields.Boolean(
+        string='Self-increase mode')
+    meter_current_value = fields.Float(
+        string='Current value',
+        help="This field can be updated manually or "
+             "daily adding a value defined.")
+    meter_autoinc_value = fields.Float(
+        string='Value to self-increase per day')
+    meter_autoinc_unit = fields.Many2one(
+        'uom.uom',
+        string='Unit of Value to self-increase per day')
+    meter_autoinc_maxvalue = fields.Float(
+        string='Maximum amount since the last revision to generate another')
```

Comment:
This could be a great `help` attribute, and use a briefer `string` one.

Reply:
You're right. I'll do it.